pkgs/development/compilers/graalvm/graalvm-oracle/default.nix

```diff
@@ -4,12 +4,12 @@
   fetchurl,
   graalvmPackages,
   useMusl ? false,
-  version ? "23",
+  version ? "24",
```

Comment:
Should the default be 21 as it is the current LTS?

Reply:
That makes sense. But maybe once we get to 25, we can keep that as the default until 29 is released.